grr/gui/api_client/connectors/http_connector.py

```diff
@@ -56,7 +56,16 @@ class ResponseFormat(object):
       routing.Rule(
           "/api/clients/<client_id>/flows/<path:flow_id>/results/files-archive",
           methods=["GET"],
-          endpoint="GetFlowFilesArchive")
+          endpoint="GetFlowFilesArchive"),
+      routing.Rule(
+          "/api/users/me/approvals/client/<client_id>",
+          methods=["POST"],
+          endpoint="RequestApproval")
```

Comment:
Please use "CreateClientApproval". The names should correspond to the names here: https://github.com/google/grr/blob/master/grr/gui/api_call_router.py#L713